AdobeStockImageApi/Api/SerializeImageInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImageApi\Api;
+
+use Magento\Framework\Api\Search\Document;
+
+/**
+ * Class SerializeImageInterface
+ * @api
```

Comment:
@sivaschenko should we marked it as an `@api`?

Reply:
`SerializeImage` is an internal operation of `AdobeStockImage` module, I don't think we need an interface for this class at all

Reply:
Aha, got it. So, if we need this serialization can we declare `imageAttributesSerializer` as a parameter in di.xml for the `AddAdobeStockImageDetailsPlugin` and `GetRelatedImages`?